tuna/miopen/miopen_lib.py

```diff
@@ -100,6 +100,12 @@ def parse_args(self):
                         dest='reset_interval',
                         required=False,
                         help='Restart interval for job in hours.')
+    parser.add_argument(
+        '--gpu_lim',
+        dest='gpu_lim',
+        type=int,
+        default=None,
```

Comment:
Would having a default be a good idea?

Reply:
In most cases we wouldn't want to impose a cap on the number of GPUs.
Though I suppose if the default was set to the integer equivalent of  infinite, then the if statement later would not have to check for None.